File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break
    else:
        print("Pick a number between 1 and 8, inclusive!")


for i in range(n):
    for space in range(n - i):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Pyramid size: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        for (int space = 0; space < n - i - 1; space++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}